### ISSUE TITLE
About training loss curve

### ISSUE THREAD

**Issue by sen-ye** [Pepper-lll/LMforImageGeneration#2]
Hello, thanks for your nice work. I'm interested in the performance of different subcode numbers under the same model size. Could you release the training loss curve of different tokenizer(2-12, 3-8, 4-6) under the same model size? Thank you.

**Comment by Pepper-lll** [comment]
> Hello, thanks for your nice work. I'm interested in the performance of different subcode numbers under the same model size. Could you release the training loss curve of different tokenizer(2-12, 3-8, 4-6) under the same model size? Thank you.

Okay, of course. I will release it in a few days.

**Comment by Doraemonzzz** [comment]
<img width="1692" alt="image" src="https://github.com/user-attachments/assets/dce3cd5e-f073-4bee-98c7-6d661e91763f">

I tried GPT-L with a tokenizer of 1-16, and the loss curve is roughly as follows (about 38 epochs were run). Is this loss reasonable? @Pepper-lll 


> Hello, thanks for your nice work. I'm interested in the performance of different subcode numbers under the same model size. Could you release the training loss curve of different tokenizer(2-12, 3-8, 4-6) under the same model size? Thank you.

**Comment by Pepper-lll** [comment]
> <img alt="image" width="1692" src="https://private-user-images.githubusercontent.com/35402766/381904936-dce3cd5e-f073-4bee-98c7-6d661e91763f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NDA5MDMsIm5iZiI6MTczMDQ0MDYwMywicGF0aCI6Ii8zNTQwMjc2Ni8zODE5MDQ5MzYtZGNlM2NkNWUtZjA3My00YmVlLTk4YzctNmQ2NjFlOTE3NjNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAxVDA1NTY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2MDk2NGE2NTViZmU4MDQ4ODBkZTM0ZDkwNWUwZGNkMThhNWM0MmU0NjU0N2Q0MWY3ZmQ3YTE3YWE3Y2QyNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jZd_ZICZluWEjg_8wQ01IQev_1w8L0zyOM99MoxjUmY">
> I tried GPT-L with a tokenizer of 1-16, and the loss curve is roughly as follows (about 38 epochs were run). Is this loss reasonable? @Pepper-lll
> 
> > Hello, thanks for your nice work. I'm interested in the performance of different subcode numbers under the same model size. Could you release the training loss curve of different tokenizer(2-12, 3-8, 4-6) under the same model size? Thank you.

This is reasonable. We have discussed the training loss and the possible reason behind it in our paper, you may have a look :>

**Comment by Doraemonzzz** [comment]
Thank you for your response. I tested the FID (without CFG) of the model after training for 50 epochs, and it was approximately 83.5. Is this reasonable? (I'm not sure if my testing is correct because I noticed that the paper does not report results without CFG)."

**Comment by Pepper-lll** [comment]
> Thank you for your response. I tested the FID (without CFG) of the model after training for 50 epochs, and it was approximately 83.5. Is this reasonable? (I'm not sure if my testing is correct because I noticed that the paper does not report results without CFG)."

We tested the 50k-FID without cfg at 400 epochs, it's around 13 for L model with tokenizer 2-10.

**Comment by Doraemonzzz** [comment]
Thank you for your response. It seems there are some issues on my side. If there are any new results, I will update them below this issue.

**Comment by Doraemonzzz** [comment]
I retested the fid of wo cfg, and the value is 15.58 (50 epoch). It seems to be a reasonable value.

**Comment by delveintodetail** [comment]
> I retested the fid of wo cfg, and the value is 15.58 (50 epoch). It seems to be a reasonable value.

Thank you for your work.

**Comment by Pepper-lll** [comment]
> I retested the fid of wo cfg, and the value is 15.58 (50 epoch). It seems to be a reasonable value.

Thanks for your information! I have released the FID result for L, XL, and XXL AR models with the most suitable tokenizers separately. I think your result is reasonable, although slightly different from mine.
